ts/src/index.ts

```diff
@@ -53,10 +53,10 @@ function hasService(config: Config):
  * Exported for testing purposes.
  */
 export async function initConfig(config: Config): Promise<ProfilerConfig> {
-  config = common.util.normalizeArguments(null, config);
+  const normalizedConfig: Config = common.util.normalizeArguments(null, config);
```

Comment:
Why is this change made? Is this related? Description says nothing about other changes, so I expected to only see the environment variable change.

Reply:
Thanks -- I've updated the description to explain this.